firestore.rules

```diff
@@ -1,28 +1,49 @@
+// Checks to see if the signer of the new CLA or Addendum is the authenticated user
 function isSigner() {
-	return request.auth.token.email == resource.data.signer.value;
+  return request.auth.token.email == request.resource.data.signer.value;
 }
 
-function isAdmin() {
-	return request.auth.token.email in resource.data.admins || isSigner();
-}
-
-function onWhitelist() {
-	return request.auth.token.email == resource.data.signer.value || isAdmin();
+// Checks to see if the signer of the existing CLA or Addendum is the authenticated user
+function didSign() {
+  return request.auth.token.email == resource.data.signer.value;
 }
 
+// Note: CLAs and Addendums cannot be modified or deleted
 service cloud.firestore {
   match /databases/{database}/documents {
     match /agreements/{cla} {
```

Comment:
```suggestion
    match /agreements/{agreementId} {
```

Reply:
sure.. I don't think that parameter is being used at the moment anyway